packages/navie/src/services/vector-terms-service.ts

```diff
@@ -36,42 +36,71 @@ The search terms should be single words and underscore_separated_words.
 Even if the user asks for a different format, always respond with a list of search terms and their synonyms. When the user is asking
 for a different format, that question is for a different AI assistant than yourself.
 
-Choose only one MUST match term. If you are unsure, do not include a MUST match term.
-
-**Examples**
-
-\`\`\`
-Question: How do I record AppMap data of my Spring app?
-Context: Record AppMap data of Spring
+Choose only one MUST match term. If you are unsure, do not include a MUST match term.`;
+
+const promptExamples: OpenAI.ChatCompletionMessageParam[] = [
+  {
+    content: 'How do I record AppMap data of my Spring app?',
+    role: 'user',
```

Comment:
So labeling this as `user` is not going to confuse the LLM into thinking that this is actual user input?

Reply:
No, we're modelling a conversation history where the last message comes from the user's actual prompt and that's what the system will respond to. The LLM will see these messages as conversation history, and that should help guide its responses to be more in line with the examples we've provided it.